MobileClient/README.md

```diff
@@ -71,10 +83,13 @@ ionic cordova platform remove android
 ionic cordova platform add android@6.4.0
 ```
 
+If you get errors like "Error occurred during initialization of VM. Could not reserve enough space for 2097152KB object heap", make sure your jvm is in the correct architecture. 
```

Comment:
Add between parenthesis what is the correct architecture or how to check this.

Reply:
```suggestion
For Windows, if you get errors like "Error occurred during initialization of VM. Could not reserve enough space for 2097152KB object heap", make sure your jvm is in the correct architecture. You can check your _system type_ in System Information APP. For example, you may see _x64-based PC_ for your system type. Then you will need to download _Windows x64 Java SE Development Kit_ from [JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
```